### PR TITLE
Adds SkyWalking agent back to dev branch and adjust according to the 2.0 changes

### PR DESCRIPTION
**Pull request by kezhenxu94**
#6515

### CODE REVIEW

**Comment by Linkaiup** [comment]
Is this different from the previously removed code?Did you add new monitor endpoint？

**Comment by Linkaiup** [comment]
> Is this different from the previously removed code?Did you add new monitor endpoint？

@kezhenxu94

**Comment by kezhenxu94** [comment]
> Is this different from the previously removed code?Did you add new monitor endpoint？

The previous version is developed on 1.3.9, dev branch has moved to 2.0 and has a lot of code changes that I adjusted the agent in this PR, no new feature is added in this PR for now.

**Comment by Linkaiup** [comment]
> > Is this different from the previously removed code?Did you add new monitor endpoint？
> 
> The previous version is developed on 1.3.9, dev branch has moved to 2.0 and has a lot of code changes that I adjusted the agent in this PR, no new feature is added in this PR for now.

Thank you for your answer

File: ext/skywalking/dashboard/dolphinscheduler.yml [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                  "width":3,
+                  "title":"QuartzScheduler Load",
+                  "height":350,
+                  "entityType":"Endpoint",
+                  "independentSelector":true,
+                  "metricType":"REGULAR_VALUE",
+                  "metricName":"endpoint_cpm",
+                  "queryMetricType":"readMetricsValues",
+                  "chartType":"ChartLine",
+                  "currentService":"dolphinscheduler::master-server",
```
Comment (caishunfeng) [inline ext/skywalking/dashboard/dolphinscheduler.yml:44]: Is the service name for skywalking must be `dolphinscheduler::master-server` ?

Comment (kezhenxu94) [reply ext/skywalking/dashboard/dolphinscheduler.yml:44]: In this pull request we only provide the dashboard for `master-server`, users can still add more dashboard for worker or other components.

The name `DolphinScheduler::master-server` is specified via https://github.com/kezhenxu94/dolphinscheduler/blob/7392b014aa773f9253f325c63bb3bc050c4148bb/script/dolphinscheduler-daemon.sh#L99

so yes its name must be `dolphinscheduler::master-server`

**Comment by Linkaiup** [comment]
Why hasn't this pr merged yet?@kezhenxu94

**Comment by kezhenxu94** [comment]
The community is planning to migrate the private network protocol to gRPC in the near future, and with that done, the agent will get huge changes, I'd rather wait for the migration to gRPC and we could make it easy to merge this.

**Comment by Linkaiup** [comment]
All right.When will it be completed? @kezhenxu94 

I want to submit some endpoint monitor in the agent.

**Comment by kezhenxu94** [comment]
> All right.When will it be completed? @kezhenxu94
> 
> I want to submit some endpoint monitor in the agent.

Hopefully in next quarter , @CalvinKirs has more clear plan

**Pull request closed without merging**